### PR TITLE
Clone news-week 

### PR DESCRIPTION
**Pull request by Luckyaremu**
Requesting for code review for the solo bootstrap project of news-week pages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,74 @@
-# newsweek-clone
\ No newline at end of file
+# newsweek-clone
+
+![screenshot](img\screenshot.jpg)
+
+A clone of the beautifull newsweek page
```
Comment (euqueme) [inline README.md]: on line #5 is "beautiful" not "bautifull" let's correct it

File: README.md [hunk 2]
```diff
@@ -1 +1,74 @@
...
+- VSCode, Linters, Stylelint, Validator.w3
+
+## Live Demo
+
+[Live Demo Link](https://frosty-leavitt-b9b46e.netlify.com)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```
Comment (euqueme) [inline README.md]: remove these instructions from the readme template ad they have no meaning for the project

File: README.md [hunk 3]
```diff
@@ -1 +1,74 @@
...
+- Clone the repository on your local machine
+- cd into the folder, find the index.html file and open it
+
+### Install
+
+- Install VSCode or any code editor you like
+- Install Node JS and nmp(most recents versions)
+- Run this command on your terminal in order to install stylelint: npm install stylelint stylelint-config-recommended --save-dev
+
+### Usage
```
Comment (euqueme) [inline README.md]: remove the Install and Usage titles as you don't need to install VS code to run this project lines #32 to #38

File: README.md [hunk 4]
```diff
@@ -1 +1,74 @@
-# newsweek-clone
\ No newline at end of file
+# newsweek-clone
+
+![screenshot](img\screenshot.jpg)
```
Comment (euqueme) [inline README.md]: replace line #3 with the following to fix the broken image

`screenshot`

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @Luckyaremu :handshake: 

### You have done an incredible work :medal_sports:  your page looks very similar to the original and you have used bootstrap very well :ok_hand: 

### There is some work to do on the screen breakpoints and general aspects of the project :point_down: 

## Small Screens

**All articles**

- [ ] The text content (mainly the bold headings) from all the articles are too big in font please reduce it 

- Yours

image

- Original

image

**Section Positioning**

- [ ] please follow the structure for the first 4 sections in this screen size:

1. TOP STORY
2. MORE STORIES
3. CULTURE & TRAVEL
4. FEATURED STORIES

- Yours

image

- Original

image

**FEATURE STORIES**

- [ ] the articles images and the header should be smaller in height and size respectively

- Yours

image

- Original

image

**CULTURE & TRAVEL**

- [ ] each article should take 100% of the screen width

- Yours

image

- Original

image

**MORE STORIES, LATEST NEWS, and the aside articles for the U.S, World, Business, Tech & Science, Culture and Sports**

- [ ] The image for the articles of these sections should take 60% of the screen width and the content only 40%

- Yours

image

- Original

image

**IN THE MAGAZINE, EDITOR'S PICK and FEATURED SLIDE SHOW**

- [ ] this sections look good enough but you have to make them fit on the page, no horizontal scroll should appear

- Yours

image

- Original

image

**SUBSCRIBE and NEWSLETTER**

- [ ] the images and content should be centered on the page yours are justified at the left (and overlapping at some points) also the NEWSLETTER image should take 100% of the screen width

- Yours

image

- Original

image

**Footer**

- [ ] the copyright and social icons should come after the NEWSWEEK logo in separate rows

- Yours

image

- Original

image

## Small and Medium Screens

**Navbar**

- [ ] The logo has to be smaller, currently is taking the whole screen making your page has a horizontal scroll also separate the SIGN IN link from the SUBSCRIBE >  button (do this also for the Big screens)

- Yours

image

- Original

image

**U.S, World, Business, Tech & Science, Culture and Sports**

- [ ] These sections should also appear in these breakpoints but they are currently hidden

- Yours

image

- Original

image

## Medium Screens

**LATEST NEWS**

* [ ] the articles are overlapping let's fix them!
* [ ] the image for the articles is too big it should take only 20% of the screen width and the content 80%

- Yours

image

- Original

image

-NEWSLETTER

the image should take 20% of the screen width and the content 80%

- Yours

image

- Original

image

## General

**OPINION**

- [ ] the color of the **OPINION** title should be light gray 

**Project repository**

- [ ] Edit your pull request description and add more to it to make it more professional
- [ ] I have added some suggestions in your README file please address them :arrow_down_small: 

### That's it! :raised_hands: 

### Please :pray: do the changes :memo: and submit back for a code review when you are done :rocket: 

Happy coding! :woman_technologist: 

Any question? reach to me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Comment by Luckyaremu** [comment]
Made changes required
1. Changes the font size
2.resized the containers
3. fixed the footer.
4. updated the readme and screenshot.
5. makes the project looks more presentable

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Lucky!  
### Great job so far :+1: 

Still there are some more changes required, before this pull request can be approved:
* [ ] Pull Request should have a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
### Mobile version
* [ ] As @euqueme  pointed earlier,  bold headings are too big, please make them smaller, so that your project will look more like original.

### Mobile and Tablet versions
* [ ] Small horizontal scrollbar is still present, that should be fixed:
image 

* [ ] Next part looks a bit messed up, please fix that:
image
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by Luckyaremu** [comment]
# Cloned Newsweek website home page

1. Image width solved
2. horizontal bar removed
3. font size reduced

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi lucky, good work improving your project:clap:, but**

1. You haven't fixed the horizontal scroll earlier pointed out. Try to find out which section of the page or sizing is causing it and fix it.

2.  The top story section on small screen doesn't look good anymore.
image

Please, take the image out of the parent div to fix this issue. The image should be a direct child of the card class. Check the bootstrap doc for Cards and ensure you use it properly. Same goes for the Editor's picks section.

**Optional:** At around 759px, your More Stories section doesn't look good. You can try to fix that also.
image

_**Kindly attend to the above and submit another code review request.**_

**Good luck:stars:**

**Comment by Luckyaremu** [comment]
# Cloned Newsweek website home page
1. horizontal bar removed
2. div more image width fixed

**Comment by davidmrtz-dev** [comment]
Pull Request under review by  **[TSE] David Elí**. 👨‍💻

**Comment by davidmrtz-dev** [comment]
### **Status: Changes required.** 🛑 👀 🔍 
****
### Hi Team. 😄 
### Very good job so far, I notice you do a great job implementing the changes required by the previous TSE 🎊 
**Before move on let's solve a couple of issues.**  💯 👨‍💻  

****

- [ ] **Horizontal scroll on medium size devices.**

As the previous TSEs mentioned, on medium-sized devices there is a horizontal scroll bar that should not be there.

Screen Shot 2020-02-22 at 3 34 14 PM
 (image from [bootstrap doc](https://getbootstrap.com/docs/4.1/layout/overview/)).

This is how it looks. ⬇️ 

Screen Shot 2020-02-22 at 3 26 26 PM

Here's how it looks on the original web page. ⬇️ 

Screen Shot 2020-02-22 at 3 26 31 PM
****
### Optional:

- [ ] **Section improve.**
In the small device's size. You can improve your divs to look more similar to the original page. 😄 💯 

This is how it looks. ⬇️ 

Screen Shot 2020-02-22 at 3 21 52 PM

Original Page. ⬇️ 

Screen Shot 2020-02-22 at 3 22 04 PM

****

### **Please kindly make the changes required and submit a new code review once you are ready!** 🙏 

Happy coding! 😄 💻  
If you have any questions or need help, you can message me on slack (@ David Eli Martinez Garcia).

**[TSE:  David Elí]** 👨‍💻 
Github: [@davitomix](https://github.com/davitomix)
Twitter: [@davitomix](https://twitter.com/davitomix007)
Linkedin: [David Elí Martínez García](https://www.linkedin.com/in/davideligarcia/)

**Comment by Luckyaremu** [comment]
Cloned Newsweek website home page
1. horizontal bar removed from all width
2. the more image and text flex throughout the page depending on the width, at less than 600px, some text goes under the image and at 600px above they stand alone which is also like the original.
one

**Comment by davidmrtz-dev** [comment]
### **Status: Approved.** ✅ 
### **Congragulations!** 🎉 
****
### Hi Team. 😄 

Great Job 💯  @Luckyaremu  @mosesogwo , you can move onto the next project. 🎊 
Happy coding! 😄 💻  

**[TSE:  David Elí]** 👨‍💻 
Github: [@davitomix](https://github.com/davitomix)
Twitter: [@davitomix](https://twitter.com/davitomix007)
Linkedin: [David Elí Martínez García](https://www.linkedin.com/in/davideligarcia/)

**Pull request merged**